R/preprocess.R

```diff
@@ -25,8 +25,8 @@
 #' @export
 #' @examples
 #' obs <- data.frame(
-#'  reference_date = as.Date("2021-01-01"),
-#'  report_date = as.Date("2022-01-01"), x = 1:10
+#'   reference_date = as.Date("2021-01-01"),
```

Comment:
@seabbs one e.g. of roxygen change. not sure if this is linter complaint or not.

Reply:
@nikosbosse ?

Reply:
Apparently some auto-linting done by my VSCode 🙈